### PR TITLE
update release-1.22 config.toml for release 1.25

### PR DESCRIPTION
**Pull request by cathchu**
update release-1.22 config.toml for new release 1.25

Version were updated according to [latest patch releases](https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md)

/cc @kcmartin @didicodes @sethmccombs @krol3 @reylejano

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview:
- https://deploy-preview-36177--k8s-v1-22.netlify.app/releases/
- https://deploy-preview-36177--k8s-v1-22.netlify.app/releases/version-skew-policy/

File: config.toml [hunk 1]
```diff
@@ -179,40 +179,40 @@ js = [
...
+githubbranch = "v1.25.0"
+docsbranch = "main"
+url = "https://kubernetes.io"
+
+[[params.versions]]
+fullversion = "v1.24.4"
 version = "v1.24"
-githubbranch = "v1.24.0"
+githubbranch = "v1.24.4"
 docsbranch = "main"
```
Comment (sftim) [inline config.toml:192]:
```suggestion
docsbranch = "release-1.24"
```

**Review by kcmartin — APPROVED** [review]

**Pull request merged**